src/components/Keypad/Keypad.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Keypad from './Keypad';
+
+describe('Keypad', () => {
+    let wrapper;
+    beforeEach(() => {
+        wrapper = shallow(
+            <Keypad
+                callOperator={jest.fn()}
+                numbers={[]}
+                operators={[]}
+                setOperator={jest.fn()}
+                updateDisplay={jest.fn()}
+            />
+        );
+    });
+
+    it('should render a <div />', () => {
+        expect(wrapper.find('div').length).toEqual(1);
```

Comment:
```suggestion
        expect(wrapper.find('div').length).toBe(1);
```
Just to be strict like Miss Honey.